suite-native/app/src/navigation/routes.ts

```diff
@@ -21,11 +23,13 @@ const receiveStack = enhanceTabOption({
     routeName: AppTabsRoutes.ReceiveStack,
     label: 'Receive',
     iconName: 'arrowLineDown',
+    focusedIconName: 'arrowLineDownFilled',
```

Comment:
That focused icon looks really ugly IMHO I would keep it same as unfocused

Reply:
I kinda agree. Let's see what has @shenkys to say.

Reply:
Fixed.